main.py

```diff
@@ -222,31 +230,56 @@ def change_visibility(region_id, image_id, public=False):
 #
 # Returns a JSON doc of the response from the API
 def delete_image(file_path, check_tag_key=None, check_tag_value=None):
```

Comment:
Let's rename this to `delete_images()` to be more accurate about what the function is doing